libs/core/src/scss/base/_functions.scss

```diff
@@ -112,6 +113,18 @@
   @return $classes;
 }
 
+/// Remove the unit of a length
+/// @param {Number} $number - Number to remove unit from
+/// @return {Number} - Unitless number
+/// Source: https://css-tricks.com/snippets/sass/strip-unit-function/
+@function strip-unit($number) {
+  @if meta.type-of($number) == 'number' and not math.is-unitless($number) {
+    @return $number / ($number * 0 + 1);
+  }
+
+  @return $number;
+}
+
```

Comment:
Is this still needed?

Reply:
Oops sorry, was too quick. Just got a notification, and thought you had requested a review, but it was just changes to this PR. I'll wait until you request a review.

Reply:
You may be right. I'll just check if it's still being used.

Reply:
Nope. It's not needed after things got cleaned up.

I _do_ think it's a very useful function to have. Should we just leave it? We can create a small issue for adding it instead, if you prefer.

Reply:
I've removed it and created [a new issue](https://github.com/kirbydesign/designsystem/issues/2162)